### PR TITLE
.rultor.yml: Install dependencies

### PR DESCRIPTION
**Pull request by Nosferatul**
Added apt-get install, apt-get update, pip3 install.

Fixes https://github.com/coala-analyzer/coala/issues/1904

### CODE REVIEW

File: .rultor.yml [hunk 1]
```diff
@@ -15,4 +18,4 @@ release:
   script: bash .misc/release.sh
 
 decrypt:
-  rultor_secrets.sh: "repo/.misc/rultor_secrets.sh.asc"
+  rultor_secrets.sh: "repo/.misc/rultor_secrets.sh.asc"
```
Comment (sils1297) [inline .rultor.yml]: Hi,

it looks like you're missing a railing newline before the end of the file here. [This article](http://stackoverflow.com/a/5813359/3212182) has more information about this issue.

File: .rultor.yml [hunk 2]
```diff
@@ -1,5 +1,8 @@
 install:
   - if ! pip3 -V ; then wget -O - https://bootstrap.pypa.io/get-pip.py | python3; fi
+  - sudo apt-get update -y
+  - sudo apt-get install espeak libclang1-3.4 python3-gi python3-dbus -y
+  - pip3 install setuptools twine wheel munkres3 coverage pylint language-check pytest tox appdirs
```
Comment (sils1297) [inline .rultor.yml]: hm this'll make the merge a lot slower I guess... also can we use the default python container then instead?

Comment (sils1297) [inline .rultor.yml]: other opinions?

File: .rultor.yml (hunk 1 above)
Comment (Adrianzatreanu) [inline .rultor.yml]: yup, newline definetely needed here!

File: .rultor.yml [hunk 3]
```diff
@@ -1,5 +1,8 @@
 install:
   - if ! pip3 -V ; then wget -O - https://bootstrap.pypa.io/get-pip.py | python3; fi
+  - sudo apt-get update -y
```
Comment (Adrianzatreanu) [inline .rultor.yml]: if this is what you meant @sils1297 , this part would take a lot indeed.

**Comment by Nosferatul** [comment]
Sorry, but I didn't understand, how can I use the default python container?
Also, what should be done with `sudo apt-get update -y` ?

**Comment by srisankethu** [comment]
@Nosferatul 2 commits ? why not do in one !

**Comment by Nosferatul** [comment]
My bad, I will fix it!

**Comment by srisankethu** [comment]
@Nosferatul Check the commit message too :+1:

**Comment by Adrianzatreanu** [comment]
Commit message should be in present tense: ``Adds x, adds y``. Also, it should include reasoning, as in why did you do this.

**Comment by Adrianzatreanu** [comment]
unack b6b870d

**Comment by Adrianzatreanu** [comment]
this is probably because there's no newline between header and body; also, it should have a limit of 80 chars per line :)

**Comment by Adrianzatreanu** [comment]
also, this is for singular. you are looking for ``these`` here :D

File: .rultor.yml [hunk 4]
```diff
@@ -15,4 +19,4 @@ release:
   script: bash .misc/release.sh
 
 decrypt:
-  rultor_secrets.sh: "repo/.misc/rultor_secrets.sh.asc"
+  rultor_secrets.sh: "repo/.misc/rultor_secrets.sh.asc"
```
Comment (sils1297) [inline .rultor.yml]: uhm there's trailing whitespace here :/

File: .rultor.yml [hunk 5]
```diff
@@ -1,5 +1,9 @@
 install:
   - if ! pip3 -V ; then wget -O - https://bootstrap.pypa.io/get-pip.py | python3; fi
+  - sudo python setup.py build
+  - sudo python setup.py install
+  - sudo apt-get install espeak libclang1-3.4 python3-gi python3-dbus -y
+  - pip3 install setuptools twine wheel munkres3 coverage pylint language-check pytest tox appdirs
```
Comment (sils1297) [inline .rultor.yml]: shouldn't you rather install the stuff from requirements.txt?

Also we don't need espeak and clang e.g. those are only needed for coala=bears right?

**Comment by Nosferatul** [comment]
I understood something else, I will check it again!

File: .rultor.yml (hunk 5 above)
Comment (Adrianzatreanu) [inline .rultor.yml]: yeah, i think only ``pip install -r requirements.txt`` should suffice here.

Comment (Adrianzatreanu) [inline .rultor.yml]: that's why we keep that file

File: .rultor.yml (hunk 4 above)
Comment (Adrianzatreanu) [inline .rultor.yml]: trailing whitespace here indeed, but don't forget to place a newline after this!

Comment (sils1297) [inline .rultor.yml]: see http://stackoverflow.com/a/5813359/3212182 about the trailing newline issue, this one is about trailing spaces right now, before you hadd the missing newline before EOF

File: .rultor.yml [hunk 6]
```diff
@@ -1,5 +1,6 @@
 install:
   - if ! pip3 -V ; then wget -O - https://bootstrap.pypa.io/get-pip.py | python3; fi
+  - pip install -r requirements.txt
```
Comment (SanketDG) [inline .rultor.yml]: Shouldn't this be `pip3`?

Comment (Nosferatul) [inline .rultor.yml]: Done!

**Comment by Adrianzatreanu** [comment]
grammar issue in commit message: ``installs``

**Comment by Adrianzatreanu** [comment]
im waiting for the commit message change and im merging this, labeling as approved so it doesn't show as pending review.

**Comment by Nosferatul** [comment]
Done!

**Comment by Adrianzatreanu** [comment]
ack 684726a

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**